_includes/releases/v22.1/v22.1.12.md

```diff
@@ -0,0 +1,104 @@
+## v22.1.12
+
+Release Date: December 12, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-1-12-general-changes">General changes</h3>
+
+- Bulk operations now log sanitized connection destinations, for example:
+
+	 `backup planning to connect to destination gs://test/backupadhoc?AUTH=specified&CREDENTIALS=redacted` [#92207][#92207]
+
+<h3 id="v22-1-12-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- Kafka sinks can now (optionally) be configured with a "Compression" field to the `kafka_sink_config` option. This field can be set to "none" (default), "GZIP", "SNAPPY", "LZ4", or "ZSTD". Setting this field will determine the compression protocol used when emitting events. [#91276][#91276]
+
+<h3 id="v22-1-12-operational-changes">Operational changes</h3>
+
+- Logs produced by setting an increased vmodule setting for s3_storage are now directed to the DEV channel rather than STDOUT. [#91960][#91960]
+
+- Introduced a metric (`replicas.leaders_invalid_lease`) that indicates how many replicas are raft group leaders but holding invalid leases. [#91194][#91194]
+
+<h3 id="v22-1-12-db-console-changes">DB Console changes</h3>
+
+- Changed the height of column selector, so it can hint there are more options to be selected once scrolled. [#91910][#91910]
```

Comment:
Nit, missing article before "column": 

 "Changed the height of the column selector"